src/components/Loader/components/IndeterminateLoader/IndeterminateLoader.style.tsx

```diff
@@ -0,0 +1,56 @@
+import { css, SerializedStyles } from '@emotion/core';
+import { rem } from 'polished';
+import { Theme } from 'theme';
+
+export const LoaderLine = () => (theme: Theme): SerializedStyles => css`
+  position: absolute;
+  opacity: 0.4;
+  background: ${theme.utils.getColor('lightCoolGray', 400)};
+  width: 150%;
+  height: ${rem(5)};
```

Comment:
The height is 8px but here is 5px (if we translate the rem to px)

Reply:
Also border-radius is missing

Reply:
loader line in the design doesn't have a radius
![image](https://user-images.githubusercontent.com/6433679/116086272-4eb2f680-a6a8-11eb-8384-01156786e5f4.png)

Reply:
The front should be with a radius. The current one is not the same as your screenshot.
![Screenshot_2](https://user-images.githubusercontent.com/11538983/116101734-c25c0000-a6b6-11eb-866e-90d69206a8b4.png)